client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/style.scss

```diff
@@ -179,6 +181,12 @@ $dark-green: #104e30;
 	}
 }
 
+.agency-tier-overview__top-content-right {
+	@include break-wide {
+		margin-inline-start: 12px;
```

Comment:
I’m considering that using a gap within `.agency-tier-overview__top-content @include break-wide` might be a better option and would eliminate the need to create a separate CSS rule for this div box.

Reply:
We already have a gap, which is being used for inline space. So, that wouldn't work here.